department-of-veterans-affairs/caseflow
# Will/appeals 17530

Resolves [APPEALS-17530](https://vajira.max.gov/browse/APPEALS-17530)

# Description
Please explain the changes you made here.

1. Added `#create_issues_update_task` to the `AppealsController`
2. Moved task creation logic away from `#update` and into `#create_subtasks!` helper method

## Acceptance Criteria

## Testing Plan
1. As user BVADWISE, intake a new case and add 4 issues (with no mst or pact on any).
2. Search the case by vet number and click on "Correct Issues."
3. Edit the first issue by checking the MST checkbox, adding "Why was this change made?" note and clicking "Save."
4. Edit the second issue by checking the Pact checkbox, adding "Why was this change made?" note and clicking "Save."
5. Edit the third issue by checking both MST and Pact checkboxes, adding "Why was this change made?" note for both, and clicking "Save."
6. Do not edit the 4th issue.
7. Click Save on the Edit Issues Page.
8. On the Case Details Page, in the Case Timeline, there should be entries for the 3 edited issues, and no entry for the 4th unedited issue.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentationa) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
